packages/react-client/src/index.ts

```diff
@@ -41,5 +53,3 @@ export type {
   AuthErrorReason,
   Encoding,
 } from "@fishjam-cloud/ts-client";
-
-export { FishjamClient } from "@fishjam-cloud/ts-client";
```

Comment:
@kamil-stasiak @czerwiukk Is there reason to keep this class exported? We have `useFishjamClient` hook.